### ISSUE TITLE
Bug: Dracula theme causes MBP fans to go crazy

### ISSUE THREAD

**Issue by sweenish** [dracula/tmux#32]
### Describe the bug
If the theme is active, my MBP's fans go nuts.

### To Reproduce
1. Follow instructions to install Dracula theme to tux
2. Run tmux
3. Wait

### Expected behavior
tmux with Dracula theme without killing the laptop

### Screenshots
<img width="1847" alt="Screen Shot 2020-05-27 at 3 01 14 PM" src="https://user-images.githubusercontent.com/18041229/83066621-f522f180-a02a-11ea-916a-a30354ea2fb5.png">

### System
 - OS: macOS 10.15.3
- Tmux Version: 3.1b from homebrew

The screenshot shows tmux fully occupying the CPU for a couple seconds about once a minute. My fans, however, never let up once they start spinning. Disabling the theme is enough to keep my laptop quiet while using tmux. I have also tried disabling the weather, wifi, and power status, but my laptop will continue to ramp up the fans.

Here's my current tmux.conf:
```
# List TPM plugins
set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'tmux-plugins/tmux-sensible'
# set -g @plugin 'dracula/tmux'

# Customize Plugins
## Dracula
# set -g @dracula-show-weather false
# set -g @dracula-show-network false

#Initialize TMUX Plugin Manager. Keep this line at the very bottom
run -b '~/.tmux/plugins/tpm/tpm'
```

**Comment by danerwilliams** [comment]
That's weird. Could you try and give some more information about your machine? For me it doesn't seem to cause any problems. I'm running with the default settings on tmux 3.0a from homebrew and macOS 10.15.3 on a 2016 mbp. 


![Screen Shot 2020-05-27 at 1 20 28 PM](https://user-images.githubusercontent.com/22798229/83068227-db7aad80-a01c-11ea-9c97-dc302be01e07.jpg)

**Comment by danerwilliams** [comment]
So that I can get more specific information on your tmux processes, can you run `ps aux | grep tmux` and send the results? This will show me more specifically the processes associated with your tmux.

**Comment by sweenish** [comment]
Output of `ps aux | grep tmux` shortly after launching it and while my fans were starting to ramp up:

```
~ 
❯ ps aux | grep tmux
sweeneya         16441  56.9  0.0  5663344    580   ??  R     3:35PM   0:00.42 tmux
sweeneya         16442  56.8  0.0  5663344    548   ??  R     3:35PM   0:00.42 tmux
sweeneya         16063   0.3  0.0  5663344   4172   ??  Ss    3:34PM   0:00.17 tmux
sweeneya         16458   0.0  0.0  4399328    728 s001  S+    3:35PM   0:00.00 grep --color=auto tmux
sweeneya         16190   0.0  0.0  4404380   1372   ??  S     3:34PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         16061   0.0  0.0  4542064   1380 s000  S+    3:34PM   0:00.01 tmux
```

**Comment by sweenish** [comment]
The screenshot you shared is what my tmux looks like the majority of the time, but it will hog most of the CPU pretty consistently, and the fans quickly ramp up and never stop.

**Comment by danerwilliams** [comment]
Weird that you have 4 tmux processes running I only have two. I don't use fish so I'm not really sure what affect that might have. My bash is the default in os x, not the version installed by homebrew.

Maybe try running tmux while in the default bash shell as well as your homebrew installed bash shell and see if there is different behavior? I don't really know what to think tbh.

**Comment by sweenish** [comment]
Using homebrew bash; output of `ps aux | grep tmux`
```
sweeneya         18991   0.4  0.0  5270128   3900   ??  Ss    3:48PM   0:00.15 tmux
sweeneya         16190   0.0  0.0  4404380   1372   ??  S     3:34PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         19345   0.0  0.0  4399328    728 s001  S+    3:49PM   0:00.00 grep tmux
sweeneya         19122   0.0  0.0  4411548   1372   ??  S     3:48PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         18989   0.0  0.0  4542064   1380 s000  S+    3:48PM   0:00.01 tmux
```

Default bash:
```
sweeneya         19866  46.3  0.0  5419632    516   ??  R     3:51PM   0:00.84 tmux
sweeneya         19864  46.3  0.0  5419632    532   ??  U     3:51PM   0:00.84 tmux
sweeneya         19865  46.2  0.0  5419632    536   ??  R     3:51PM   0:00.84 tmux
sweeneya         19863  46.2  0.0  5419632    568   ??  R     3:51PM   0:00.84 tmux
sweeneya         19543   0.0  0.0  5419632   4048   ??  Ss    3:50PM   0:00.13 tmux
sweeneya         19541   0.0  0.0  4542064   1380 s000  S+    3:50PM   0:00.01 tmux
sweeneya         19122   0.0  0.0  4411548   1372   ??  S     3:48PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         16190   0.0  0.0  4404380   1372   ??  S     3:34PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         19868   0.0  0.0  4416736    756 s001  S+    3:51PM   0:00.00 grep tmux
sweeneya         19676   0.0  0.0  4404380   1372   ??  S     3:50PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
```

Since Catalina uses zsh as its default, here's one with the system zsh:
```
sweeneya         20881  45.7  0.0  5411440    540   ??  R     3:55PM   0:01.80 tmux
sweeneya         20878  45.7  0.0  5411440    584   ??  R     3:55PM   0:01.80 tmux
sweeneya         20879  45.7  0.0  5411440    544   ??  U     3:55PM   0:01.80 tmux
sweeneya         20880  45.7  0.0  5411440    540   ??  R     3:55PM   0:01.80 tmux
sweeneya         20438   0.0  0.0  4542064   1388 s000  S+    3:55PM   0:00.01 tmux
sweeneya         19676   0.0  0.0  4404380   1372   ??  S     3:50PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         19122   0.0  0.0  4411548   1372   ??  S     3:48PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         16190   0.0  0.0  4404380   1372   ??  S     3:34PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         20884   0.0  0.0  4399328    728 s001  S+    3:55PM   0:00.00 grep tmux
sweeneya         20582   0.0  0.0  4404380   1384   ??  S     3:55PM   0:00.01 bash /Users/sweeneya/.tmux/plugins/tmux/scripts/sleep_weather.sh true
sweeneya         20440   0.0  0.0  5411440   4172   ??  Ss    3:55PM   0:00.12 tmux
```

**Comment by danerwilliams** [comment]
Weird. This behavior is really unusual to me and I am unable to replicate the bug. Seems like a good opportunity for a community PR.

**Comment by sweenish** [comment]
I wonder if you tried updating your tmux to match mine, if the dracula theme then behaves similarly.

**Comment by ginglis13** [comment]
@sweenish I'm using the same tmux version as you on a 2016 mbp and haven't had the same issues w/ CPU usage.

It looks like in the configuration you shared above you have commented out some options, including the dracula theme. Do you experience the same issues w/ CPU usage when you are using default tmux? If you run prefix-R you will reload your tmux env.

**Comment by sweenish** [comment]
The screenshots and `ps aux` output I pasted are all with the theme enabled. I currently have the theme disabled because tmux behaves just fine while the theme is disabled.

Here is `ps aux | grep tmux` output with fish and Dracula disabled.
```
sweeneya         21346   0.1  0.0  5374576   2980   ??  Ss    4:28PM   0:00.09 tmux
sweeneya         21456   0.0  0.0  4409568    752 s001  S+    4:29PM   0:00.00 grep --color=auto tmux
sweeneya         21344   0.0  0.0  4551280   1400 s000  S+    4:28PM   0:00.01 tmux
```

I waited about a minute, as by that time my fans would be going nuts with the theme enabled.

**Comment by ginglis13** [comment]
@sweenish gotcha just wanted to double check since the original screenshot shows the theme but the conf you provided shows it commented out.

**Comment by sweenish** [comment]
When I submitted this bug, it was due to the behavior of my work laptop. Having configured tmux for my personal MBP (2018 15", Catalina 10.15.5, 16 GB RAM, other specs same), with the same three plugins enabled (tpm, sensible, and Dracula), the theme is behaving just fine and my laptop has remained quiet.

I am willing to chalk this up to the management software or some other work configuration that I am powerless to change. I'll leave it up to someone with some authority to close or decide if they still want to pursue this.
